package com.example.demo.repository;

import com.example.demo.entity.Permission;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

import java.util.UUID;

public interface PermissionRepository extends JpaRepository<Permission, UUID>,
        JpaSpecificationExecutor<Permission>
{

}